CoreFarms.cs

```diff
@@ -139,15 +139,22 @@ public void HonorHall(int goldQuant = 100000000)
         Core.Logger($"Farming {goldQuant} gold using HonorHall Method");
 
         Core.RegisterQuests(3992, 3993);
+        Core.Join("honorhall", "r1", "Left");
```

Comment:
Not needed

Reply:
^

Reply:
if using the method i posted bellow. this will be required.